testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentModule.java

```diff
@@ -90,15 +89,15 @@ public EnvironmentConfig provideEnvironmentConfig(EnvironmentOptions options, En
     @Singleton
     public PortBinder providePortBinder(EnvironmentOptions options)
     {
-        switch (options.bindPorts) {
-            case -1:
-                return new PortBinder.DefaultPortBinder();
-            case 0:
-                return new PortBinder.FixedPortBinder();
-            default:
-                checkArgument(options.bindPorts > 0, "Ports base must be greater than 0");
-                return new PortBinder.ShiftingPortBinder(new PortBinder.FixedPortBinder(), options.bindPorts);
+        if (options.bindPorts) {
+            if (options.bindPortsBase > 0) {
+                return new PortBinder.ShiftingPortBinder(new PortBinder.FixedPortBinder(), options.bindPortsBase);
+            }
+
+            return new PortBinder.FixedPortBinder();
         }
+
+        return new PortBinder.DefaultPortBinder();
```

Comment:
the DefaultPortBinder could be renamed to somethng like Nonbinding....

Reply:
no change requested here

Reply:
Ports are still bound, but to a random host port (this is the default testcontainers behavior)